spaces/S000201/properties/P000042.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000201
+property: P000042
+value: true
```

Comment:
Can this be deleted as we have https://topology.pi-base.org/spaces/S000201/properties/P000043 ?

Reply:
Oh I see - it's being deduced from this plus locally hausdorff.